db/migrations/20150714121034_add_trades.js

```diff
@@ -3,11 +3,11 @@
 exports.up = function(knex, Promise) {
   return Promise.all([
     knex.schema.createTable('trades', function(table) {
-      table.uuid('id').primary().notNullable();
```

Comment:
Im curious about removing notNullable() here? What is the reasoning behind this? 
It seems to me that we would never want a trade to be created without an ID?